tests/unit_tests/ensemble_evaluator/test_ensemble_evaluator_config.py

```diff
@@ -35,3 +37,28 @@ def test_load_config(unused_tcp_port):
     assert sock is not None
     assert not sock._closed
     sock.close()
+
+
+def test_that_get_machine_name_is_predictive(mocker):
+    """For ip addresses with multiple PTR records we must ensure
+    that get_machine_name() is predictive to avoid mismatch for SSL certificates.
+
+    The order DNS servers respond to reverse DNS lookups for such hosts is not
+    defined."""
+
+    # GIVEN that reverse DNS resolution results in two names in random order:
+    ptr_records = ["foobar01.equinor.com.", "barfoo01.internaldomain.barf."]
+
+    # Whether index 0 or 1 is picked is not important, only predictiveness
+    expected_resolved_name = ptr_records[1].rstrip(".")
```

Comment:
Make a comment the this gets sorted and therefore `barfoo01.internaldomain.barf` is expected after sorting.

Reply:
fixed!